example/BUILD

```diff
@@ -0,0 +1,12 @@
+load("@proto_gen_firebase_rules//:defs.bzl", "firestore_rules_proto_library", "firestore_rules_binary")
+
+firestore_rules_proto_library(
+    name = "schema",
+    srcs = ["schema.proto"],
+)
+
+firestore_rules_binary(
+    name = "example",
+    srcs = ["main.rules"],
+    deps = [":schema"],
+)
```

Comment:
Can we get a test that these are what we would expect from the output of this rule? Happy to throw it in `integration_test.py` which has some stuff for golden file based testing. Otherwise put it in a test in this directory.